genomic_data_service/region_indexer_task.py

```diff
@@ -317,6 +317,16 @@ def list_targets(dataset):
     return target_labels
 
 
+def get_ancestry(file_metadata):
+    ancestry = None
+    aliases = file_metadata.get('aliases')
+    if aliases:
+        tag = aliases[0].split('-')[-1]
+        if not tag.startswith('PMID'):
+            ancestry = tag
```

Comment:
This will work but is SUPER fragile.  I think we should put in some detailed comments -  "This is to catch files that are in ENCSR533BHN / PMID30650056.   This may need to be updated if the metadata at ENCODE is improved